hasher-matcher-actioner/hmalib/common/reactioner_models.py

```diff
@@ -0,0 +1,73 @@
+import typing as t
+
+from dataclasses import dataclass, fields
+
+from hmalib.models import MatchMessage, BankedSignal
+from hmalib.common.logging import get_logger
+from hmalib.common.actioner_models import ActionPerformer, ActionLabel
+from hmalib.aws_secrets import AWSSecrets
+
+from threatexchange.api import ThreatExchangeAPI
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class ReactActionPerformer(ActionPerformer):
+    @property
+    def reaction(self) -> str:
+        raise NotImplementedError
+
+    def perform_action(self, match_message: MatchMessage) -> None:
+        api_key = AWSSecrets.te_api_key()
+        api = ThreatExchangeAPI(api_key)
+
+        indicator_ids = {
+            dataset_match_details.banked_content_id
+            for dataset_match_details in match_message.matching_banked_signals
+            if dataset_match_details.bank_source == "te"
```

Comment:
"te" is being extracted to a constant in [another PR](https://github.com/facebook/ThreatExchange/pull/503/files#diff-5e4a23ec0c264ec4ce29b65fe691d4db71393e2972d2803b2444d1fd020020eaR23-R24). This should use the same constant. Avoid magic values as much as possible.

Reply:
Will change once it lands